src/dbnode/topology/types.go

```diff
@@ -54,6 +54,11 @@ type HostShardSet interface {
 type Initializer interface {
 	// Init will return a new topology
 	Init() (Topology, error)
+
+	// TopologySet returns whether the topology is able to be
+	// initialized immediately or if instead it will blockingly
+	// wait to be set on initialization
+	TopologySet() (bool, error)
```

Comment:
super nit: how about `TopologyIsSet`, `TopologySet` had me thinking `a set of topologies` and I was bouncing around a little bit trying to figure out how it was used

Reply:
Sure, don't really mind - I don't usually believe "Is" as a prefix to a method is good, but I can see how as it is currently is confusing.